cmd/gaia/cli_test/test_helpers.go

```diff
@@ -288,15 +288,15 @@ func (f *Fixtures) CLIConfig(key, value string, flags ...string) {
 
 // TxSend is gaiacli tx send
 func (f *Fixtures) TxSend(from string, to sdk.AccAddress, amount sdk.Coin, flags ...string) (bool, string, string) {
-	cmd := fmt.Sprintf("../../../build/gaiacli tx send %s %s %v --from=%s", to, amount, f.Flags(), from)
+	cmd := fmt.Sprintf("../../../build/gaiacli tx send %s %s %v --from=%s --broadcast-mode=block", to, amount, f.Flags(), from)
```

Comment:
maybe we can make this configurable via `gaiacli` config and set it in `InitializeFixtures`? This would also have the benefit of allowing users to set their desired mode there as well.

Reply:
Yeah I agree. I'll update this PR to support that.

Reply:
Done!